app/assets/stylesheets/masthead.scss

```diff
@@ -0,0 +1,23 @@
+#masthead-container {
+  background-color: #9D2520;
```

Comment:
Wondering if we should create our own variables file to store colors like this (if not already available to us).

Reply:
@jvine can you clarify which red I should use for the Bento app? 
Mockups have #9D2520, but that didn't map to any brand colors or pre-existing colors in SeachWorks. Should I be using Cardinal Red or Bright Red? 

<img width="842" alt="stanford_reds" src="https://user-images.githubusercontent.com/5402927/30445179-1710864a-993a-11e7-9b96-7ae5ef22377c.png">

Reply:
Let me know about the following areas: 
(1) font color for the Search bar
(2) background color for the masthead
(3) background color for bento section border-tops

Reply:
@camillevilla Grrr omnigraffle doesn't seem to export colours accurately. The red is Cardinal throughout. Everywhere Cardinal. "Search" heading, border-tops, and the logo banner are Cardinal.

"Masthead" has a specific meaning in SW Catalog that I think is different here. 
- page background is #fbfbf9
- logo banner is #8c1515
- subnav menu bar is #797268

Does that cover it?

Reply:
Yes! Thank you~